src/test/java/Homework19.java

```diff
@@ -0,0 +1,31 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+public class Homework19 extends BaseTest{
+
+    @Test(enabled = true)
+    public void testDeletePlaylist(){
+        //
+        String namePl   = "MyPlaylist";
+        String selector = "//section[@id='playlists']//li//a[contains(text(),'"+namePl+"')]";
+        login();
+
+        //try to find my playlist
+        List<WebElement> ListPlaylist = driver.findElements(By.xpath(selector));
+
+        if (ListPlaylist.isEmpty()) {
+            createPlaylist(namePl);
+            WebElement PL = driver.findElement(By.xpath(selector));
+            deletePlaylist(PL);
+        }
+        else {
+            for (WebElement currentPlayList:ListPlaylist ) {
+                deletePlaylist(currentPlayList);
```

Comment:
**Use in test methods, the following logical split:   GIVEN – WHEN - THEN**
    
For example

```
 @Test

    public void testChangeProfileName (){
        // GIVEN

        String newName  = getRandomString();

        String currentPassword = "te$t$tudent";

        provideEmail("demo@class.com");

        providePassword();

        clickSubmitBtn();

 

        // WHEN

        openUserProfilePage();

        setName(newName);

        setCurrentPassword(currentPassword);

 

 

        // THEN

        Assert.assertTrue(getSuccessPopUp().isDisplayed());

}
```
Please, add Assert: **one test - one Assert**

Reply:
Hi, i added assert and create a new pull request.